classes/mail/variables/SubmissionEmailVariable.inc.php

```diff
@@ -25,8 +25,9 @@ class SubmissionEmailVariable extends Variable
     public const SUBMISSION_TITLE = 'submissionTitle';
     public const SUBMISSION_ID = 'submissionId';
     public const SUBMISSION_ABSTRACT = 'submissionAbstract';
+    public const AUTHORS_SHORT = 'authorsShort';
     public const AUTHORS = 'authors';
-    public const AUTHORS_FULL = 'authorsFull';
+    public const AUTHORS_WITH_AFFILIATION = 'authorsWithAffiliation';
```

Comment:
Where is this used? I didn't find it in any of Kate's rewritten emails.

Reply:
Thanks, removed that variable